tests/measure-perf.bats

```diff
@@ -190,6 +211,27 @@ function get_perf_opts() {
   [ "$npng" -eq 0 ] # no detailed charts because of --skip-perf
 }
 
+@test "measure perf --skip-perf --use-vdenc --skip-ffmpeg raw h265" {
```

Comment:
h265 vdenc is not supported on gen9, this test will fail. And h264 test will fail as well if HUC is not enabled. Please, take a look into this change which @dsocek did for quality: https://github.com/intel/media-delivery/commit/c31ea6ac907b64d359b59429af8ced0105d73b19

We need to add 2 things:
1. Check in tests whether vdenc is actually supported (that's just copy-paste from quality test)
2. Preferably if we will add similar check to msperf to error out with some nice message that the requested mode is not supported.